spec/requests/errors_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe "Errors", type: :request do
+  describe "GET /not_found" do
+    it "returns http success" do
+      get "/errors/not_found"
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe "GET /unprocessable_entity" do
+    it "returns http success" do
+      get "/errors/unprocessable_entity"
+      expect(response).to have_http_status(:success)
```

Comment:
status must be unprocessable_entity, not success

Reply:
fixed